src/main/java/com/epam/spring/homework3/repository/impl/CategoryRepositoryImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.spring.homework3.repository.impl;
+
+import com.epam.spring.homework3.model.Category;
+import com.epam.spring.homework3.repository.CategoryRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers